tests/output/ConsoleInteractionTest.py

```diff
@@ -49,6 +49,15 @@
 STR_PROJECT_WIDE = "Project wide:"
 
 
+class DummyFileProxy:
+
+    def __init__(self, lines):
+        self.lines = lines
```

Comment:
The __lines can be accessible to a static member of the class. Seems like we're talking about the same thing though ^.^